src/abstracts.ts

```diff
@@ -106,7 +107,7 @@ export interface SpritePosition {
 
 export interface BlockField {
     name: string;
-    value: any;
+    value: string | number;
```

Comment:
I think value is optional. I don't think it shows up for variables.

Reply:
I want to do a deeper dive on these fields / typedefs as kind of a matter of fact with the Scratch team and you at some point